### ISSUE TITLE
The input format of images for the resent pretrained models? RGB or BGR?

### ISSUE THREAD

**Issue by ChangErgou** [zhanghang1989/gluoncv-torch#17]
What is the image format for resent? RGB or BGR?

**Comment by zhanghang1989** [comment]
The settings are the same as PyTorch pretrained model:

All pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (3 x H x W), where H and W are expected to be at least 224. The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. You can use the following transform to normalize:

**Comment by ChangErgou** [comment]
thanks a lot. changzhonghan16 Email:changzhonghan16@mails.ucas.ac.cn Signature is customized by Netease Mail Master On 01/08/2019 04:38, Hang Zhang wrote: The settings are the same as PyTorch pretrained model: All pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (3 x H x W), where H and W are expected to be at least 224. The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. You can use the following transform to normalize: — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.
